pages/project/list/views/index.jsx

```diff
@@ -35,12 +35,12 @@ const Projects = ({
     />
     {
       allowedActions.includes('project.apply') && (
-        <p className="control-panel projects-actions">
+        <div className="control-panel projects-actions">
```

Comment:
This will likely have knock-on effects on styling because a `p` comes with inherent styles but a `div` does not.

Reply:
it does not - I checked :)